core/src/test/java/org/elasticsearch/rest/action/cat/RestTableTests.java

```diff
@@ -186,13 +186,24 @@ public void testRowOutOfBounds() {
         });
     }
 
+    public void testUnknownHeader() {
+        Table table = new Table();
+        table.startHeaders();
+        table.addCell("compare");
+        table.endHeaders();
+        restRequest.params().put("s", "notaheader");
+        expectThrows(UnsupportedOperationException.class, () -> {
```

Comment:
I think it'd be nice to do:
```java
Exception e = expectThrows(UnsupportedOperationException.class, () -> {
  RestTable.getRowOrder(table, restRequest);
});
assertEquals("blah blah blah", e.getMessage());
```

It is just nice to assert something about the message so you know *which* `UnsupportedOperationException` you get.